Makefile

```diff
@@ -16,13 +16,20 @@ dep:
 dep-up:
 	dep ensure
 
-test: build
+
+test:
+	GOOS=windows go build -o fn
```

Comment:
Why this change?  Doesn't seem to make much sense.

Reply:
because we don't know if we can compile fn until that moment when CI attempts to release fn

Reply:
I think that is the valid reason to check if fn compiles for various platforms we consider as "supported"

Reply:
I just mean why build five times, one overwriting the next. Seems like the old way was fine.

Reply:
that's the point, each next step overwrites previous and the last step will produce linux-compatible binary that CI can use for further testing.

Reply:
I don't think this is necessary and it doesn't build the correct one for whoever is running this command.

Reply:
i extracted that to its own namespace:` test-cross-compile`